sechub-pds-solutions/scancode/env-example

```diff
@@ -0,0 +1,56 @@
+# The PDS start mode
+#   debug - starts only the container
+#   localserver - starts the server locally
+PDS_START_MODE=debug, localserver
```

Comment:
Is this example syntactically correct?
I'd suggest to make it work and put only one (reasonable) option. E.g.:
`PDS_START_MODE=localserver`

(similar for the rest of the file)

Reply:
The purpose of this file is to show all possible options and values. It is intended for documentation purposes only. This file does not have to be syntactically correct.